client/wfnews-war/src/main/angular/package.json

```diff
@@ -61,6 +61,7 @@
     "@ngrx/store-devtools": "15.4.0",
     "@qqnluaq/smk": "=1.0.16000",
     "@swimlane/ngx-charts": "20.1.0",
+    "@turf/turf": "^6.5.0",
```

Comment:
Turf is already a part of SMK so you shouldn't need to import it. Also, you can import the specific modules you need, rather than the whole library